source/documentation/adrs/adr-021.html.md.erb

```diff
@@ -0,0 +1,34 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: ADR-021 Management of DNS Records through OctoDNS
+last_reviewed_on: 2024-06-25
+review_in: 6 months
+---
+
+# ADR-021 Managing DNS Records Through OctoDNS
+
+## Status
+
+✅ Accepted
+
+## Context
+
+The Operations Engineering (OE) team at the Ministry of Justice (MoJ) recognized the need for a system to manage DNS records consistently across multiple providers and teams. Currently, DNS management is performed leading to inconsistencies and a lack of traceability.
```

Comment:
This is too generic. This explains why any team might want to manage infrastructure as code. The context here should be addressing why Operations Engineering want to manage as code. This should be coveted in the associated problems statement and feedback from the Security Team.

Reply:
See [Capturing Security Requirements for DNS Estate Management Enhancement](https://github.com/ministryofjustice/operations-engineering/issues/4320)

Reply:
and [Problem Statements](https://docs.google.com/presentation/d/1nRyOex6tZL4FDOON1iltsHiIwlBcp6EUvLqYjipjI_s/edit#slide=id.g2c1986e6aeb_0_0)